### ISSUE TITLE
security/tinc: TINCVPN interface

### ISSUE THREAD

**Issue by tuxec** [opnsense/plugins#204]
Hello,

I am using opnsense 17.1.9 with tinc plugin, currently on three devices. 

On two of those devices a TINCVPN interface and "TincVPN net" were automatically created (even though under "Interfaces / Assignments" the tinc0 interface is still not assigned). However, on the third device neither TINCVPN interface nor "TincVPN net" were ceated, even though the VPN connection has been successfully established.

Any idea what to do to get the TINCVPN being created? 

Many thanks!

Petr

**Comment by mimugmail** [comment]
I remember that you have to edit a firewall rule, do nothing and click save. Would you please try it?

**Comment by tuxec** [comment]
1. Update for 17.1.9: I have just added the new not yet assigned interface tinc0 as OPT2 (three interfaces are physical on the board) under "Assignments / Interfaces / +" and this step has created the TINCVPN. After deleting the "OPT2" the TINCVPN was **not** deleted and can be found now in the "Firewall / Rules" along with "TincVPN net".

2. @mimugmail: I will try your suggestion with another installation.

3. Yesterday I installed another board with 17.1.4 and then upgraded to 17.1.10 and then installed os-tinc plugin. Unfortunately, here nothing works, no tincd server is shown in Services on Dashboard, no configuration is possible because of problem #198 when trying to add a new Network. Reboot does not improve anything. "configctl tinc list ciphers" lists correctly the ciphers, but even then nothing improves in the GUI.
